Documentation/getting-started.md

```diff
@@ -25,11 +25,19 @@ In addition, if you wish to try out authenticating against Google's OIDC backend
 
 # Create Database
 
-`createdb dex_db`
+Within the Postgres database create a database and user for dex to use (these can be named anything). This requires logging in as a user with appropirate permissions.
```

Comment:
On the PostgreSQL server, login as a user with appropriate permissions and create a database and user for dex to use. These can be named arbitrarily, but are called `dex_db` and `dex`, respectively, in this example.